src/plays/simple-live-chat/SimpleLiveChat.jsx

```diff
@@ -0,0 +1,47 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+import "./simpleLiveChat.scss";
+
+import { useState } from "react";
+import SignIn from "./components/signin-button";
+import StorageHandler from "./components/Storage";
+
+import DisplayChat from "./components/chat/chat";
+
+function SimpleLiveChat(props) {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  const { user } = props;
+  const [LoggedUser, setLoggedUser] = useState(user);
```

Comment:
The state variable name should be camel case `loggedUser`

Reply:
yes done